.github/workflows/build_library.yml

```diff
@@ -0,0 +1,39 @@
+name: Binary Release Build
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+    
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+  
+    steps:
+      - uses: actions/checkout@v2
+    
+      - name: Set up docker for build (x86_64)
+        # Add more of these steps to support other architectures
+        run: |
+          docker build \
+            --build-arg DOCKER_ARCH=amd64 \
+            -t getsentry/postgres_wal2json:build \
+            -f Dockerfile.builder .
+          docker run -d \
+            -v `pwd`:/wal2json \
+            --name postgres_wal2json \
+            -e POSTGRES_HOST_AUTH_METHOD=trust \
+            getsentry/postgres_wal2json:build \
+            -d postgres
+      - name: Build in Docker (x86-64)
+        run: |
+          docker exec \
+            postgres_wal2json \
+            make && mv wal2json.so wal2json-Linux-x86_64.so
```

Comment:
This logic should be moved into an `entrypoint` script so all we do is to run `docker run -v $(pwd):/workspace` and we "magically" get our built artifact there.

Reply:
Moved in an entrypoint script